### PR TITLE
net-dns/host: EAPI and cross compile fix

### PR DESCRIPTION
**Pull request by bjacquin**

### CODE REVIEW

**Comment by monsieurp** [comment]
I've just sent george@gentoo.org an email about this PR. He will take a look.

File: net-dns/host/host-20070128-r1.ebuild [hunk 1]
```diff
@@ -1,8 +1,8 @@
-# Copyright 1999-2013 Gentoo Foundation
+# Copyright 1999-2015 Gentoo Foundation
 # Distributed under the terms of the GNU General Public License v2
 # $Id$
 
-EAPI="2"
+EAPI=5
```
Comment (mgorny) [inline net-dns/host/host-20070128-r1.ebuild]: This ebuild has stable keywords. For EAPI bump, you must revbump it and do the changs on new revision.

Comment (mgorny) [inline net-dns/host/host-20070128-r1.ebuild]: And remember to drop to ~arch when revbumping.

File: net-dns/host/host-20070128-r1.ebuild [hunk 2]
```diff
@@ -17,26 +17,24 @@ IUSE="debug"
 # Bug 91515
 RESTRICT="test"
 
-RDEPEND=""
-DEPEND="${RDEPEND}
-	>=sys-apps/sed-4"
+DEPEND=""
+RDEPEND="${DEPEND}"
```
Comment (mgorny) [inline net-dns/host/host-20070128-r1.ebuild]: Please don't reorder this.

**Comment by mgorny** [comment]
Is there a bug open about this? If there is, you know what to do ;-).

**Comment by bjacquin** [comment]
Pull request updated as requested

**Comment by mgorny** [comment]
Upstream? ;-)

**Comment by bjacquin** [comment]
maintainer contacted

**Comment by mgorny** [comment]
Any reply? Is there an official bugtracker/ml of some kind or did you just mail him privately?

**Comment by bjacquin** [comment]
No reply yet unfortunately. I send an email privately as it seems to be no mailling according to the homepage (http://www.weird.com/~woods/projects/host.html)

**Comment by mgorny** [comment]
To be honest, I'm not sure if we're going to merge this. The package seems abandoned, and we already hack it seriously to make it work. I've filed treeclean request as https://bugs.gentoo.org/show_bug.cgi?id=564048. Let's see how that goes.

**Comment by bjacquin** [comment]
Still no answer from upstream, so I'm fine as well with a removal

**Comment by mgorny** [comment]
@bjacquin, does `dig` and `host` tools from bind-tools suit your needs?

**Comment by bjacquin** [comment]
They does, as long as busybox as well. net-dns/host is the tiniest host command, however I'm fine to switch to something else and prefer to use something maintained

**Comment by mgorny** [comment]
Ok then. I'm going to close this PR for now and see how things go. In any case, please keep the branch so we can reopen at a later time.

As a side suggestion, please note that the common `host` implementations usually fail to use system resolver. On glibc systems, you may prefer using getent, e.g.:

```
$ host pomiot.local
Host pomiot.local not found: 3(NXDOMAIN)
$ getent hosts pomiot.local
fe80::210:a7ff:fe27:7364 pomiot.local
$ getent ahosts pomiot.local
192.168.1.3     STREAM pomiot.local
192.168.1.3     DGRAM  
192.168.1.3     RAW    
fe80::210:a7ff:fe27:7364 STREAM 
fe80::210:a7ff:fe27:7364 DGRAM  
fe80::210:a7ff:fe27:7364 RAW

**Pull request closed without merging**